doc/user/nhrpd.rst

```diff
@@ -68,7 +80,60 @@ command defines the GRE subnet):
 Configuring NHRP
 ================
 
-FIXME
+.. index::  ip nhrp holdtime (1-65000)
+.. clicmd:: ip nhrp holdtime (1-65000)
+
+	    Holdtime is the number of seconds that have to pass before stopping to
+	    advertise an NHRP NBMA address as valid. It also controls how often
+	    NHRP registration requests are sent. By default registrations are sent
+	    every one third of the holdtime.
+
+.. index::  ip nhrp map A.B.C.D|X:X::X:X A.B.C.D|local
+.. clicmd:: ip nhrp map A.B.C.D|X:X::X:X A.B.C.D|local
+
+	    Map an IP address of a station to the station's NBMA address.
+
+.. index::  ip nhrp network-id (1-4294967295)
+.. clicmd:: ip nhrp network-id (1-4294967295)
+
+	    Enable NHRP on this interface and set the interface's network ID.
+	    The network ID is used to allow creating multiple nhrp domains
+	    on a router when multiple interfaces are configured on the router.
+	    Interfaces configured with the same ID are part of the same
+	    logical NBMA network. The ID is a local only parameter and is not
+	    sent to other NHRP nodes and so IDs on different nodes do not need
+	    to match. When NHRP packets are received on an interface they are
+	    assigned to the local NHRP domain for that interface.
```

Comment:
Spaces not tabs please, all the doc formatting stuff is documented in the developer manual